build.sbt

```diff
@@ -18,7 +18,7 @@ lazy val Versions = new {
   val typesafeConfig    = "1.3.1"
 }
 
-def assemblyExcludeLocal(names: String*) =
+def assemblyExcludeLocal(names: Seq[String]*) =
```

Comment:
I think we should write in Scaladoc what `Seq[String]` is for, i.e. this is the module and name selector, i.e. `Seq("servicediscovery", "javadsl")`

Reply:
I agree - I'll get this in the next PR as I don't want to rebase for this.

Reply:
Ok